### ISSUE TITLE
Feature Request: Please add config option to require a --confirm flag for production broker

### ISSUE THREAD

**Issue by cameronbraid** [birdayz/kaf#92]
I would like to be able to configure my kaf config for my production cluster so that it requires me to run --config for any destructive operations

Say I run 

kaf topic delete topic-a

and I am using my dev config, I would expect this to run without requiring a --confirm

However if I have forgotten to switch from the prod config to dev, I would like that to fail saying that a --confirm flag is required.

I'd hate to accidentally delete a few GB of data !

**Comment by birdayz** [comment]
i agree, we could add a flag to the config entry of a cluster: "requireConfirmation"for destructive operations.
